### PR TITLE
move simpletests to examples/tests directory

### PR DESCRIPTION
**Pull request by apahim**
There's no reason for having a separated simpletests directory, let's move
them to the tests directory.

Reference: https://trello.com/c/VFcxxwIp
Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

**Comment by ldoktor** [comment]
Yep, it sounds good but due to network issues I can't rebuild the packages. I'll resume on this in the afternoon.

File: avocado.spec [hunk 1]
```diff
@@ -110,7 +110,6 @@ examples of how to write tests on your own.
 
 %files examples
 %{_datadir}/avocado/tests
-%{_datadir}/avocado/simpletests
 %{_datadir}/avocado/wrappers
```
Comment (ldoktor) [inline avocado.spec]: you should also update the %changelog, am I right, @clebergnu?

**Comment by ldoktor** [comment]
yep, it looks good and works fine. I'm not sure about the changelog, if it's OK to update it only per release, or whether it's necessary to update it with each modification. From the log I think it's the later.

**Comment by clebergnu** [comment]
The RPM spec suggests that each change increments the release number, and I have tried to stick to that rule.  Not that I always remember to do so.  BTW, it also helps with testing the new package because it becomes an upgrade if you have the previous versions.

@apahim would you mind bumping the release number and adding a changelog entry?

**Comment by apahim** [comment]
Ok, no problem.

**Pull request closed without merging**